packages/terra-card/src/Card.scss

```diff
@@ -1,10 +1,15 @@
+@import '~terra-mixins';
+@import './mixins';
+@import './themes';
+
 :local {
   /* stylelint-disable selector-class-pattern  */
   .card {
     background-color: var(--terra-card-background-color, #fff);
     border: var(--terra-card-border, 1px solid #c8cacb);
     border-radius: var(--terra-card-border-radius, 3px);
     box-shadow: var(--terra-card-box-shadow, none);
+    margin: var(--terra-card-margin, 0);
```

Comment:
Remove margin property.  This should be controlled by whatever component lays them out.